custom_enumerable_methods.rb

```diff
@@ -0,0 +1,117 @@
+module Enumerable
+  def my_each
+    return to_enum(:my_each) unless block_given?
+
+    my_self = self
+    i = 0
+    while i < my_self.size
+      yield(my_self[i])
+      i += 1
+    end
+    my_self
+  end
+
+  def my_each_with_index
```

Comment:
When a block is given and when it is a range it should return the range itself after calling the given block once for each element in self. [Reference](https://ruby-doc.org/core-2.6.5/Enumerable.html) 
Consider the following example:
![Screen Shot 2020-07-30 at 9 15 29 AM](https://user-images.githubusercontent.com/52430951/88933620-46c34480-d245-11ea-8225-6d61b84f27cc.png)

Reply:
# Thank You! :bouquet: 
## Above issue is fixed now! :heavy_check_mark: 

## I helped us to learn more and fix the things.